rulebook.md

```diff
@@ -103,7 +103,7 @@ The qualification rounds would be held on the side of the main arena.
 
 The qualification starting line is a marked 1m wide section of the pool wall from where the AUV should be deployed from. The AUV **has to touch the wall** at the begining of the run.
 
-At approximately 10m from the starting line the qualification gate is hanging from the surface of the water. The AUV has to go through the gate to qualify.
+At approximately 10m from the starting line the qualification gate is hanging from the surface of the water. The AUV has to go through the gate to qualify. If the AUV touches ANY 3 sides of the qualification gate, the attempt is considered a failure.
```

Comment:
Sounds like it has to touch 3 sides at the same time. You mean "any of the 3 sides"?